docs/quickstarts/fastify.mdx

```diff
@@ -27,171 +64,166 @@ After following this guide, you should have a working Fastify app with public an
   })
 
   const start = async () => {
     try {
-      await fastify.listen({ port: 3000 })
-    } catch (err) {
-      fastify.log.error(err)
+      await fastify.listen({ port: 8080 })
+    } catch (error) {
+      fastify.log.error(error)
       process.exit(1)
     }
   }
 
   start()
   ```
 
-  ### Require authentication for a route
-
-  To protect a route using Clerk, you can use `getAuth` to check if the user is authenticated. If the user is not authenticated, you can return a 403 error.
+  To protect a route using Clerk, use the `getAuth()` method to check if the user is authenticated. If not, it returns an error response. If authenticated, it retrieves the user information using `clerkClient`.
 
   ```ts {{ filename: 'index.ts' }}
   fastify.get('/protected', async (request, reply) => {
-    const { userId } = getAuth(request)
-    if (!userId) {
-      return reply.code(403).send()
-    }
+    try {
+      const { userId } = getAuth(request)
 
-    const user = await clerkClient.users.getUser(userId)
-    return { user }
+      if (!userId) {
+        return reply.code(403).send({ error: 'Unauthorized request.' })
+      }
+
+      const user = await clerkClient.users.getUser(userId)
+
+      return reply.send({
+        message: 'Authentication state retrieved successfully.',
+        user,
+      })
+    } catch (error) {
+      fastify.log.error(error)
+      return reply.code(500).send({ error: 'Failed to retrieve user information.' })
+    }
   })
   ```
 </Steps>
 
-### Using Clerk for specific routes only
+### Configure `clerkPlugin` for specific routes
 
-If you want to use Clerk for specific routes only, you can register the plugin for specific routes. In this example, the routes are split into two groups: one for public routes and one for private routes.
+If you want to use Clerk for specific pages only, you can register the plugin for specific routes. In the following example, the routes are split into public and private routes.
 
 ```ts {{ filename: 'index.ts' }}
-import * as dotenv from 'dotenv'
-
-dotenv.config()
-
+import 'dotenv/config'
 import Fastify, { FastifyPluginCallback } from 'fastify'
 import { clerkClient, clerkPlugin, getAuth } from '@clerk/fastify'
 
 const fastify = Fastify({ logger: true })
 
-/**
- * Register Clerk only for a subset of your routes
- */
-const protectedRoutes: FastifyPluginCallback = (instance, opts, done) => {
+const protectedRoutes: FastifyPluginCallback = (instance, options, done) => {
   instance.register(clerkPlugin)
+
   instance.get('/protected', async (request, reply) => {
     const { userId } = getAuth(request)
     if (!userId) {
-      return reply.code(403).send()
+      return reply.code(403).send({ message: 'Access denied. Authentication required.' })
     }
 
     const user = await clerkClient.users.getUser(userId)
+
+    // Only authenticated users will see the following message
+    reply.send({ message: 'This is a protected route.' })
     return { user }
   })
```

Comment:
is this supposed to be in this format:

`reply.send({ message, user })`

without the `return { user }` ??